lib/modules/manager/vendir/schema.ts

```diff
@@ -16,7 +22,8 @@ export const HelmChart = z.object({
 
 export const Contents = z.object({
   path: z.string(),
-  helmChart: HelmChart,
+  helmChart: HelmChart.optional(),
+  git: GitRef.optional(),
```

Comment:
are both allowed? if not we should create two different objects.

Reply:
ok, You misunderstood. are helm chart and git are allowed at same time on same object?

```yaml
  - path: XYZ
    helmChart: ...
    git: ...
```

Reply:
Ah, no, two different objects. Okay, I'll update.

Reply:
https://github.com/renovatebot/renovate/blob/b33ce14e88756dcc7e50bfb032529c238e7d0c92/lib/modules/platform/github/issue.ts#L27

use a union type 🙃

Reply:
Updated to use a union type